pkg/services/apiserver/builder/helper.go

```diff
@@ -154,6 +157,7 @@ func InstallAPIs(
 	storageOpts *options.StorageOptions,
 	reg prometheus.Registerer,
 	kvStore grafanarest.NamespacedKVStore,
+	cfg *setting.Cfg,
```

Comment:
Can we avoid passing all of config down here? It looks like we could set it as a storage option like we do for the dualwrite mode in config.go

Reply:
Good catch! I will change the code according to your suggestion

Reply:
done!